### ISSUE TITLE
I found maybe something wrong in the code

### ISSUE THREAD

**Issue by tigerdhl** [HuCaoFighting/Swin-Unet#135]
val_loader = DataLoader(db_train, batch_size=batch_size, shuffle=False, num_workers=args.num_workers,
                            pin_memory=True,
                            worker_init_fn=worker_init_fn)

maybe the "db_train" is wrong , "db_val" is right

**Comment by aminreza-nutrien** [comment]
Hello, I have noticed the same thing. I believe instead of having db_train over there, we should have db_val.
